src/util/activity-tracker/admin.ts

```diff
@@ -90,37 +78,35 @@ export async function adminLook(interaction: ChatInputCommandInteraction): Promi
 					(activityTrackerBlacklistDb.get("general-user")?.includes(user.id) ? "disabled" : "enabled") +
 					"`\nBlacklisted games: No game is blacklisted. Every game gets logged"
 			);
-		embed = addDefaultEmbedFooter(embed);
+		embed = addEmbedColor(embed);
 		await interaction.reply({ embeds: [embed], ephemeral: true });
 		return;
 	}
 
 	const str = "`" + blacklist.join("`, `") + "`";
 
-	let embed = new EmbedBuilder()
+	const embed = new EmbedBuilder()
 		.setTitle(`${user.tag}'s blacklist`)
 		.setDescription(
 			"Trackstatus: `" +
 				(activityTrackerBlacklistDb.get("general-user")?.includes(user.id) ? "disabled" : "enabled") +
 				"`\nBlacklisted games: " +
 				str
 		);
-	embed = addDefaultEmbedFooter(embed);
-	await interaction.reply({ embeds: [embed], ephemeral: true });
+	await interaction.reply({ embeds: [addEmbedColor(embed)], ephemeral: true });
 }
 
 export async function adminShow(interaction: ChatInputCommandInteraction) {
 	const blacklist: string[] | undefined = activityTrackerBlacklistDb.get("general-game");
 
 	if (!blacklist || blacklist.length == 0) {
 		let embed = new EmbedBuilder().setTitle("The global blacklist is empty...");
-		embed = addDefaultEmbedFooter(embed);
+		embed = addEmbedFooter(embed);
```

Comment:
yeah we could but I think doing that isn't worth it when in the end we would just do `embed.addEmbedFooter()` instead of `addEmbedFooter(embed)`

If you want to you can add it. Don't have anything against it. Just isn't worth it for me :)

Reply:
Well I was thinking that just constructing it would do it automatically so you don't have to run anything extra

Reply:
Oh okay then we would need two extensions for each footer ig.
But like I said it doesn't bother me.

Reply:
WDYM? Is there more than one function for adding footers to embeds??

Reply:
Yes ^^ That's why I didn't do it in the first place ig.
There are different kinds of footers.

Reply:
Something that annoys me for a long time is the requirement to do `embed = addEmbedFooter(embed)`
The function just returns the object you give in, so if Im not mistaking, just `addEmbedFooter(embed)` should work fine which I think is much better. And if required could add a options `returns` which will return the object in case it will be used while sending the embed

Reply:
Actually not. You just always did this
```ts
let embed = new EmbedBuilder()
	.setTitle("Removed a game from the global blacklist!")
	.setDescription(`Removed "${game}" from global blacklist.`);
embed = addEmbedColor(embed)
await interaction.reply({ embeds: [embed], ephemeral: true });
```
Instead of just this
```ts
const embed = new EmbedBuilder()
	.setTitle("Removed a game from the global blacklist!")
	.setDescription(`Removed "${game}" from global blacklist.`);
await interaction.reply({ embeds: [addEmbedColor(embed)], ephemeral: true });
```
I tried to correct it everywhere but ig I just missed it there.